CI task failed on pre-commit

The pre-commit step always failed on tf-gpu-1 runner

``` text
Run pre-commit run -a --show-diff-on-failure
/home/github/actions-runner/_work/_temp/d83aab53-811f-4e5c-bb09-802e5f50be22.sh: line 1: pre-commit: command not found
Error: Process completed with exit code 127.
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -26,6 +26,7 @@ jobs:
       shell: bash
       run: |
         set -ex
+        echo $PATH
         pre-commit run -a --show-diff-on-failure
     - name: Build Dev Docker
       shell: bash

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -15,7 +15,7 @@ jobs:
   # This workflow contains a single job called "greet"
   CUDA10-TF115:
     # The type of runner that the job will run on
-    runs-on: [self-hosted, gpu-t4-1]
+    runs-on: [self-hosted, gpu-t4]
       
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
@@ -70,7 +70,6 @@ jobs:
       uses: actions/checkout@v2.4.0
     - name: Build Dev Docker
       run: |
-        exit 1
         set -e
         git submodule sync
         git submodule update --depth=1 --init --recursive
@@ -111,7 +110,6 @@ jobs:
       uses: actions/checkout@v2.4.0
     - name: Build Dev Docker
       run: |
-        exit 1
         set -e
         git submodule sync
         git submodule update --depth=1 --init --recursive
@@ -152,7 +150,6 @@ jobs:
       uses: actions/checkout@v2.4.0
     - name: Build Dev Docker
       run: |
-        exit 1
         set -e
         git submodule sync
         git submodule update --depth=1 --init --recursive

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -25,8 +25,7 @@ jobs:
     - name: pre-commit
       shell: bash
       run: |
-        set -ex
-        echo $PATH
+        export PATH=$HOME/.local/bin:$PATH
         pre-commit run -a --show-diff-on-failure
     - name: Build Dev Docker
       shell: bash

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -15,7 +15,7 @@ jobs:
   # This workflow contains a single job called "greet"
   CUDA10-TF115:
     # The type of runner that the job will run on
-    runs-on: [self-hosted, gpu-t4]
+    runs-on: [self-hosted, gpu-t4-1]
       
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
@@ -25,6 +25,7 @@ jobs:
     - name: pre-commit
       shell: bash
       run: |
+        set -ex
         pre-commit run -a --show-diff-on-failure
     - name: Build Dev Docker
       shell: bash
@@ -69,6 +70,7 @@ jobs:
       uses: actions/checkout@v2.4.0
     - name: Build Dev Docker
       run: |
+        exit 1
         set -e
         git submodule sync
         git submodule update --depth=1 --init --recursive
@@ -109,6 +111,7 @@ jobs:
       uses: actions/checkout@v2.4.0
     - name: Build Dev Docker
       run: |
+        exit 1
         set -e
         git submodule sync
         git submodule update --depth=1 --init --recursive
@@ -149,6 +152,7 @@ jobs:
       uses: actions/checkout@v2.4.0
     - name: Build Dev Docker
       run: |
+        exit 1
         set -e
         git submodule sync
         git submodule update --depth=1 --init --recursive

